inc/classes/CDN/CDN.php

```diff
@@ -61,11 +61,18 @@ public function rewrite_srcset( $html ) {
 		if ( ! preg_match_all( $pattern, $html, $srcsets, PREG_SET_ORDER ) ) {
 			return $html;
 		}
-
 		foreach ( $srcsets as $srcset ) {
 			$sources    = explode( ',', $srcset['sources'] );
+			$sources    =
+				array_unique(
+					array_map(
+						function( $source ) {
+							return trim( $source );
+						},
+						$sources
+					)
+				);
```

Comment:
This code can be simplified into a single line:

```php
$sources  = array_unique( array_map( 'trim', $sources ) );
```

PHP looks up `trim`. As it's a callback, it executes that function. [See `array_map` in the PHP manual](https://www.php.net/manual/en/function.array-map.php) for more info.

Why do this?

- It's more readable.
- It'll run slightly faster and take a little less memory. Why? PHP doesn't have to create the closure in memory.